cmd/podman/images/build.go

```diff
@@ -241,8 +241,12 @@ func buildFlagsWrapperToOptions(c *cobra.Command, contextDir string, flags *buil
 	}
 
 	pullPolicy := imagebuildah.PullIfNewer
-	if c.Flags().Changed("pull") && !flags.Pull {
-		pullPolicy = imagebuildah.PullIfMissing
+	if c.Flags().Changed("pull") {
+		if flags.Pull {
+			pullPolicy = imagebuildah.PullAlways
+		} else {
+			pullPolicy = imagebuildah.PullNever
+		}
```

Comment:
This does not comply to our docs.  We claim to do a "PullIfMissing" here in the man pages and a blog post or two.

Yes, we differ from Docker here, but personally I think it's a better difference the way it had just changed to.   Regardless, if we want to exactly match Docker, this change doesn't do it.  `docker build --pull=false` does a PullIfMissing, not PullNever.  

docker build  => PullIfMissing
docker build --pull => PullAlways
docker bulid --pull = true => PullAlways
docker build --pull = false => PullIfMissing

Reply:
Ok can you fix the --o pullifmissing side of this.